media/js/firefox/australis/common.js

```diff
@@ -51,6 +51,12 @@
             }).on('click', function(e) {
                 e.preventDefault();
 
+                // Track sync button clicks
+
+                dataLayer.push({
```

Comment:
@jpetto - Are you wanting to have `window.` appended to all the `dataLayer.push` calls?